internal/config/config.go

```diff
@@ -55,6 +56,7 @@ type Config struct {
 	Runs []*Run `json:"runs"`
 
 	DockerRegistriesAuth map[string]*DockerRegistryAuth `json:"docker_registries_auth"`
+	TaskTimeout          *time.Duration                 `json:"task_timeout"`
```

Comment:
Will this parse human format duration format supported by time.ParseDuration (like 15s, 5m etc...)?